source/gameengine/Rasterizer/RAS_OpenGLRasterizer/RAS_StorageVao.h

```diff
@@ -24,6 +24,7 @@
 #define __RAS_STORAGE_VAO_H__
 
 #include "RAS_AttributeArray.h"
+#include "RAS_DisplayArray.h"
```

Comment:
It looks very odd to fix a linker error by including an header, but anyway I don't have the faith to look deeper and I think that you already done it right, thanks.

Reply:
Yeah, very very strange. I did so much tests during a week trying a lot things without any luck except for this solution,